frontend/src/app/layouts/register/registration-routing.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
```

Comment:
What you need registration-routing.module.ts for ? Path can be written to app-routing.module.ts